php72/Dockerfile

```diff
@@ -0,0 +1,149 @@
+FROM centos:7
+
+# Install base packages.
+RUN yum -y install epel-release centos-release-scl yum-plugin-ovl deltarpm && \
+    yum -y update && \
+    yum -y install sudo ssh curl less vim-minimal dnsutils openssl
+
+RUN yum-config-manager --enable rhel-server-rhscl-7-rpms
+
+# Download confd.
+ENV CONFD_VERSION 0.11.0
+RUN curl -L "https://github.com/kelseyhightower/confd/releases/download/v$CONFD_VERSION/confd-$CONFD_VERSION-linux-amd64" > /usr/bin/confd && \
+    chmod +x /usr/bin/confd
+ENV CONFD_OPTS '--backend=env --onetime'
+
+RUN yum -y install \
+      https://rpms.remirepo.net/enterprise/remi-release-7.rpm \
+    yum -y update
+
+# Add the IUS repository. This is needed for git2.
+RUN curl -L "https://centos7.iuscommunity.org/ius-release.rpm" > /usr/local/ius-release.rpm && \
+    rpm -Uvh /usr/local/ius-release.rpm
+
+RUN yum -y install \
+    # Install PHP
+    php72 \
+    php72-php-devel \
+    php72-php-gd \
+    php72-php-xml \
+    php72-php-pdo \
+    php72-php-mysql \
+    php72-php-mysqlnd \
+    php72-php-mbstring \
+    php72-php-fpm \
+    php72-php-opcache \
+    php72-php-pecl-memcache \
+    php72-php-pecl-xdebug \
+    php72-php-posix \
+    php72-php-mcrypt \
+    php72-php-pecl-yaml \
+    php72-php-pecl-zip \
+    # Install Ruby
+    ruby193 \
```

Comment:
I don't. If the pattern lab/particle stuff we're using has fully swapped over to libsass it may be that we don't need it at all. Perhaps we should put out a call for comments on whether anyone thinks it's still valuable in the base image.